fuzzyfinder.go

```diff
@@ -263,6 +264,67 @@ func (f *finder) _draw() {
 	}
 }
 
+func parseColor(rs *[]rune) (tcell.Color, bool, error) {
+	// only parses for 16 colors
+	re, errCompile := regexp.Compile(`\[[0-9;]*m`)
```

Comment:
Maybe, we can reduce calls of `regexp` functions.

``` go
s := `\[31mfoo`

re := regexp.MustCompile(`(\\\[[0-9;]+m).*`)
subStrs re.FindSubmatch([]byte(s))
if len(subStrs) == 0 {
  // no matching
}

fmt.Println(subStrs[1]) // \[31m
fmt.Println(strings.TrimPrefix(s, string(subStrs[1]))) // foo
fmt.Println(strings.TrimSuffix(strings.TrimPrefix(string(subStrs[1]), `\[`), "m")) // 31
```

regexps for each color can also be replaced with the compare the colors.

``` go
switch n {
case "30":
  return tcell.ColorBlack, ...
case "31":
  return tcell.ColorRed, ...
...
}
```

Reply:
Thanks for the feedback! I've pushed a new commit with the changes you suggested.